csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala

```diff
@@ -90,10 +90,9 @@ trait MetaDataValidator {
         CSVParser.DEFAULT_QUOTE_CHARACTER
     }
 
-    //TODO CSVReader does not appear to be RFC 4180 compliant as it does not support escaping a double-quote with a double-quote between double-quotes
```

Comment:
Thank you for your review!

The reason that I have to delete the comment, 
In my understanding, 
If the escaped double quotation marks such as [\"], CSVReader supports double quotation marks. (Isn't this based upon RFC 4180?)

For example the following CSV is possible to verify.

<*>CSV Schema
version 1.0
@totalColumns  2 @separator ',' @quoted
Name:
Age:

<*>CSV data
"This is \"OK\"","10"


<*>Remarks
Backslash must be escaped.
[\] -> [\\]